### ISSUE TITLE
Support custom profile

### ISSUE THREAD

**Issue by Berrysoft** [nabijaczleweli/cargo-update#282]
A default custom profile could be defined in `~/.cargo/config.toml`, for example,
```toml
[profile.release-lto]
inherits = "release"
lto = true
codegen-units = 1
```
I would like to configure cargo-update to use a custom profile for specific installed crate.

**Comment by nabijaczleweli** [comment]
Can you try the current `master` branch (at least 13101d57253989e421c09619629cde216af4ac8f)? `cargo install-update-config --build-profile release-lto $package` should work (and `cargo install-update ...` should respect it if you gave it to `cargo install $package`).

**Comment by Berrysoft** [comment]
Seems that it works, thanks!

**Comment by nabijaczleweli** [comment]
Great, released in [`v16.0.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v16.0.0)!
